core/src/main/java/tech/pegasys/web3signer/core/Runner.java

```diff
@@ -124,6 +127,12 @@ public void run() {
         routerFactory.addGlobalHandler(LoggerHandler.create(LoggerFormat.DEFAULT));
       }
 
+      routerFactory.addGlobalHandler(
+          CorsHandler.create(buildCorsRegexFromConfig())
+              .allowedHeader("*")
+              .allowedHeader("content-type")
+              .allowedMethod(HttpMethod.DELETE));
```

Comment:
don't we need the other `HttpMethod` here as well? POST/GET/OPTIONS

Reply:
The OPTIONS, POST and GET seemed to provided by default. I had to add DELETE for that to work with my testing.

Reply:
I added the others for clarity in the code